### ISSUE TITLE
How to restore bundled plugins?

### ISSUE THREAD

**Issue by BuzzKnightbeer** [qbittorrent/search-plugins#130]
I deleted the bundled plugins and can't find a way to get them back without completely reinstalling the program and loosing all my settings. I'm particularly looking for the Jacket plugin.

Is there a way to get the bundled plugins separately? They are not listed on the normal plugin site.

**Comment by ngosang** [comment]
Close qBittorrent, remvoe the plugins's folder and start qBittorrent:

    Windows: %localappdata%\qBittorrent\nova3
    Linux: ~/.local/share/data/qBittorrent/nova3
    OS X: ~/Library/Application Support/qBittorrent/nova3

**Comment by ngosang** [comment]
Yes, you can delete `nova3` folder

**Comment by BuzzKnightbeer** [comment]
I tried deleting it twice but the plugins are still not back. Do you maybe have a different approach? I'm on Windows btw.

**Comment by ngosang** [comment]
Deleting the folder `nova3` and restarting qBittorrent should work. If it doesn't reinstall qBittorrent.

**Comment by jprokos** [comment]
I am seeing this same issue on Mac. I am told there are no search plugins installed.
I looked in ```~/Library/Application Support/qBittorrent``` and saw two directories one named ```nova``` and another named ```nova3```

I removed ```nova3``` as stated and restarted qBittorent. ```nova3``` was re-created but there are still no search engines in Search.

If it helps I have attached the contents of both ```nova``` and ```nova3``` on my Mac.

[Archive.zip](https://github.com/qbittorrent/search-plugins/files/4399718/Archive.zip)
